scripts/cluster-install/install.py

```diff
@@ -0,0 +1,392 @@
+        print(stdout.read())
+        time.sleep(0.5)  # Avoiding running over the print by the next logging
+
+
+def _get_vm(instances_details, instance, key_path, vm_username):
+    private_ip = str(instances_details[instance][0])
+    public_ip = str(instances_details[instance][1])
+    return VM(private_ip, public_ip, instance, key_path, vm_username)
+
+
+def _get_instances_dict(instances_details, key_path, vm_username):
+    instances_dict = {}
+    factory = _get_vm(instances_details, 'factory', key_path, vm_username)
+    db1 = _get_vm(instances_details, 'postgresql_1', key_path, vm_username)
+    db2 = _get_vm(instances_details, 'postgresql_2', key_path, vm_username)
+    db3 = _get_vm(instances_details, 'postgresql_3', key_path, vm_username)
+    rabbit1 = _get_vm(instances_details, 'rabbitmq_1', key_path, vm_username)
+    rabbit2 = _get_vm(instances_details, 'rabbitmq_2', key_path, vm_username)
+    rabbit3 = _get_vm(instances_details, 'rabbitmq_3', key_path, vm_username)
+    manager1 = _get_vm(instances_details, 'manager_1', key_path, vm_username)
+    manager2 = _get_vm(instances_details, 'manager_2', key_path, vm_username)
+    manager3 = _get_vm(instances_details, 'manager_3', key_path, vm_username)
+    instances_dict['postgresql_nodes'] = [db1, db2, db3]
+    instances_dict['rabbitmq_nodes'] = [rabbit1, rabbit2, rabbit3]
+    instances_dict['manager_nodes'] = [manager1, manager2, manager3]
+    instances_dict['factory'] = [factory]
+    if len(instances_details) > 10:
+        load_balancer = _get_vm(instances_details, 'load_balancer', key_path,
+                                vm_username)
+        instances_dict['load_balancer'] = [load_balancer]
+    return instances_dict
+
+
+def _install_load_balancer(instances_dict, key_path):
+    logging.info('Installing load balancer')
+    load_balancer = instances_dict['load_balancer'][0]
+    managers_details = ''
+    scp_local_to_remote(key_path, load_balancer, LOCAL_INSTALL_CLUSTER_CERTS,
+                        REMOTE_PARENT_DIRECTORY)
+    scp_local_to_remote(key_path, load_balancer, 'install_load_balancer.sh',
+                        REMOTE_PARENT_DIRECTORY)
+    for manager in instances_dict['manager_nodes']:
+        managers_details += (manager.private_ip + ' ' +
+                             manager.public_ip + ' ')
+    install_command = 'sudo bash {script_path} {managers_details} {directory}' \
+        .format(script_path=os.path.join(REMOTE_PARENT_DIRECTORY,
+                                         'install_load_balancer.sh'),
+                managers_details=managers_details,
+                directory=REMOTE_PARENT_DIRECTORY)
+    stdin, stdout, stderr = load_balancer.exec_command(install_command)
+    print(stdout.read())
+    return
+
+
+def _create_install_cluster_directory(license_path, download_link):
+    logging.info('Creating `install_cluster` directory')
+    os.system('rm -rf {}_old'.format(LOCAL_INSTALL_CLUSTER))
```

Comment:
`shutil.rmtree` is a more direct way to do this ;)

Reply:
Done, thanks.